stix_shifter_modules/vision_one/stix_transmission/results_connector.py

```diff
@@ -0,0 +1,31 @@
+from stix_shifter_utils.modules.base.stix_transmission.base_results_connector import BaseResultsConnector
+from stix_shifter_utils.utils import logger
+from stix_shifter_utils.utils.error_response import ErrorResponder
+
+
+class ResultsConnector(BaseResultsConnector):
+    def __init__(self, api_client):
+        self.api_client = api_client
+        self.logger = logger.set_logger(__name__)
+
+    def create_results_connection(self, search_id, offset, length):
+        try:
+            min_range = offset
+            if length > 1000:
+                self.logger.warning("The length exceeds length limit. Use default length: 1000")
```

Comment:
is this warning necessary?

Reply:
Users should be informed that if users set length to be greater than 1000, the API will use 1000 instead.

Reply:
so does it mean no matter what the user sets the length of the query it will only return 1000 results?

Reply:
If the user sets the length less than or equals to 1000, the query will return the amount of data that the user wants. If the user set the length greater than 1000, the query will return 1000 results only.